src/PhpBrew/Command/EnvCommand.php

```diff
@@ -16,7 +16,7 @@ public function arguments($args)
         $args->add('installed php')
             ->optional()
             ->validValues(function () {
-                return \PhpBrew\Config::getInstalledPhpVersions();
+                return array_merge(\PhpBrew\BuildFinder::findMatchedBuilds(), array(''));
```

Comment:
The argument is already optional, so adding an empty string as a valid value seems redundant. Furthermore, it will be taken into account during completion which is undesirable.

It looks like the behavior of passing an empty string as an argument is specific to Fish:
```
$ php -r 'echo $_SERVER["argc"];' --
1

$ php -r 'echo $_SERVER["argc"];' -- $UNDEFINED
1

$ set VERSION ""
$ php -r 'echo $_SERVER["argc"];' -- $VERSION
2
```
While Bash behaves in a more expected way:
```
$ VERSION=

$  php -r 'echo $_SERVER["argc"];' -- $VERSION
1
```
But the Fish script doesn't take this behavior into account:

https://github.com/phpbrew/phpbrew/blob/4c93f44deb8fcdec04efecfb8c66c09547d2b37e/shell/phpbrew.fish#L350-L354